install/empty_data.php

```diff
@@ -9235,6 +9235,7 @@ public function getEmptyData(): array
             ],
         ];
 
+        // allowed_statuses is set using default value, @see install/mysql/glpi-empty.sql:7578 ( table `glpi_tickettemplates` )
```

Comment:
```suggestion
        // allowed_statuses is set using default value, @see install/mysql/glpi-empty.sql ( table `glpi_tickettemplates` )
```
The correct line number most likely won't be maintained as the file changes.